catt/cli.py

```diff
@@ -255,7 +259,8 @@ def cast(settings, video_url, subtitle, force_default, random_play, no_subs):
         if cst.info_type == "url":
             cst.play_media_url(stream.playlist_entry_url,
                                title=stream.playlist_entry_title,
-                               thumb=stream.playlist_entry_thumbnail)
+                               thumb=stream.playlist_entry_thumbnail,
+                               content_type = content_type or "video/mp4")
```

Comment:
Why not make sure higher up that this exists, and avoid checking again here and farther down?

Reply:
Because I could not find a way to write it more clearly, provided that this check is made only in the url case.